float64 ack
---
int32[] id_sent
float64[] tvec_sent
float64[] rvec_sent
